prov/rxm/src/rxm_cq.c

```diff
@@ -418,11 +418,9 @@ static ssize_t rxm_lmt_send_ack(struct rxm_rx_buf *rx_buf)
 	tx_entry->context 	= rx_buf;
 	tx_entry->tx_buf 	= tx_buf;
 
-	tx_buf->pkt.ctrl_hdr.version    = OFI_CTRL_VERSION;
 	tx_buf->pkt.ctrl_hdr.type 	= ofi_ctrl_ack;
-	tx_buf->pkt.ctrl_hdr.conn_id 	= rx_buf->conn->handle.remote_key;
+	tx_buf->pkt.ctrl_hdr.conn_id	= rx_buf->conn->handle.remote_key;
 	tx_buf->pkt.ctrl_hdr.msg_id 	= rx_buf->pkt.ctrl_hdr.msg_id;
-	tx_buf->pkt.hdr.version         = OFI_OP_VERSION;
```

Comment:
Can you clarify why it's okay to drop setting these values as part of the commit message?

Reply:
These values aren't checked/used anywhere. I think we can refuse to set these values until they aren't used.

Reply:
As a wire protocol, we need to set these and not leave them assigned to random values.  Otherwise, we have no hope of providing compatibility with wire protocol changes.  If the versions are set to 0, and the memory buffers are set to 0, then we might be okay here.  We could add an assert in that case.  Or we can pre-set these values at creation time.

The receive side should verify the version as part of normal packet processing.  If that isn't being done, we should fix that rather than leave version unset.

Reply:
@shefty Thanks for explaining this.
I've dropped this change for now.
I'll fix this in the separate PR.